### PR TITLE
small_cleanup

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
CI on PR fails because of the CI dependency bug.

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,74 +33,30 @@ generate_messages(
...
-    include/tue/serialization/archive.h
-    include/tue/serialization/conversions.h
-    include/tue/serialization/output_archive.h
-    include/tue/serialization/filesystem.h
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
 )
 
+file(GLOB_RECURSE HEADER_FILES include/*.h)
```
Comment (ar13pit) [inline CMakeLists.txt:48]: Can be removed. Seems like it is not being used anywhere

File: package.xml [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+<?xml-model
+  href="http://download.ros.org/schema/package_format3.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>tue_serialization</name>
+  <version>0.0.0</version>
+  <description>The tue_serialization package</description>
 
-	<maintainer email="s.v.d.dries@tue.nl">Sjoerd van den Dries</maintainer>
+  <maintainer email="s.v.d.dries@tue.nl">Sjoerd van den Dries</maintainer>
```
Comment (ar13pit) [inline package.xml]: Add the current maintainer

**Review by ar13pit — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (MatthijsBurgh) [reply CMakeLists.txt:48]: By defining this, QtCreator will show these files.

Comment (ar13pit) [reply CMakeLists.txt:48]: Okay.

**Review by ar13pit — APPROVED** [review]

**Pull request merged**